Firestore/Source/Remote/FSTDatastore.h

```diff
@@ -221,14 +229,14 @@ NS_ASSUME_NONNULL_BEGIN
              snapshotVersion:(FSTSnapshotVersion *)snapshotVersion;
 
 /**
- * Called by the FSTWatchStream when the underlying streaming RPC is closed for whatever reason,
- * usually because of an error, but possibly due to an idle timeout. The error passed to this
- * method may be nil, in which case the stream was closed without attributable fault.
+ * Called by the FSTWatchStream when the underlying streaming RPC is interrupted for whatever
+ * reason, usually because of an error, but possibly due to an idle timeout. The error passed to
+ * this method may be nil, in which case the stream was closed without attributable fault.
  *
  * NOTE: This will not be called after `stop` is called on the stream. See "Starting and Stopping"
  * on FSTStream for details.
  */
-- (void)watchStreamDidClose:(NSError *_Nullable)error;
+- (void)watchStreamWasInterrupted:(NSError *_Nullable)error;
```

Comment:
Naming: `watchStreamWasInterruptedWithError:` (the old name was wrong too)

Reply:
Done